### DISCUSSION TITLE
Live connection from excel ? [May 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2248] [Discord Threads]
> 🙋‍♂️ **Tastier Sub**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Like you would like to connect to nocodb to via excel ?  
#  
> 🙋‍♂️ **Tastier Sub**  
#### Correct.  
#  
> 💁 **o1lab**  
#### not possible  
#  
> 💁 **o1lab**  
#### would you be able to connect to a 3rd party api in excel ?  
#  
> 💁 **o1lab**  
#### if so you could do possibly do that  
#  
> 🙋‍♂️ **Tastier Sub**  
#### I'll have to look into it. It looks like folks have been able to connect Airtable through some code - I was hoping something similar existed for NocoDB.   
####    
#### https://community.airtable.com/t/airtable-link-to-excel-via-power-query/18675  
#  
> 💁 **o1lab**  
#### pls give it a shot & let us know  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979082345913151598)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
